scripts/roles/todd/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+- include_vars: toddvars.yml

```

Comment:
Because this is an Ansible role and `toddvars.yml` exists in `todd/vars` you should not need to add the `include_vars` here. As it should pick them up as part of the role itself.